patches/0058-LLDP-firmware-version.patch

```diff
@@ -0,0 +1,66 @@
+From 490eaef533d16c8a01a6188851d1c7f7a84a363a Mon Sep 17 00:00:00 2001
+From: Owen Anderson <owenthomasanderson@gmail.com>
+Date: Fri, 30 Jul 2021 16:30:59 -0400
+Subject: [PATCH] Changed LLDP description to include firmware version and
+ br-wan ip
+
+Signed-off-by: Owen Anderson <owenthomasanderson@gmail.com>
+---
+ package/network/services/lldpd/Makefile         |  2 +-
+ package/network/services/lldpd/files/lldpd.init | 13 ++++++++-----
+ 2 files changed, 9 insertions(+), 6 deletions(-)
+
+diff --git a/package/network/services/lldpd/Makefile b/package/network/services/lldpd/Makefile
+index 5b36155b71..eb1fe6539d 100644
+--- a/package/network/services/lldpd/Makefile
++++ b/package/network/services/lldpd/Makefile
+@@ -9,7 +9,7 @@ include $(TOPDIR)/rules.mk
+ 
+ PKG_NAME:=lldpd
+ PKG_VERSION:=1.0.3
+-PKG_RELEASE:=1
++PKG_RELEASE:=2
+ 
+ PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+ PKG_SOURCE_URL:=https://media.luffy.cx/files/lldpd
+diff --git a/package/network/services/lldpd/files/lldpd.init b/package/network/services/lldpd/files/lldpd.init
+index 64dfccfebe..fd8347cd9f 100644
+--- a/package/network/services/lldpd/files/lldpd.init
++++ b/package/network/services/lldpd/files/lldpd.init
+@@ -12,11 +12,8 @@ LLDPD_CONFS_DIR=/tmp/lldpd.d
+ 
+ find_release_info()
+ {
+-	[ -s /etc/os-release ] && . /etc/os-release
+-	[ -z "$PRETTY_NAME" ] && [ -s /etc/openwrt_version ] && \
+-		PRETTY_NAME="$(cat /etc/openwrt_version)"
+-
+-	echo "${PRETTY_NAME:-Unknown OpenWrt release} @ $(cat /proc/sys/kernel/hostname)"
++	FW_VERSION=`sed -n 's/FW_IMAGE_ACTIVE:\(.*\)/\1/p' < /usr/opensync/.versions`
++	echo "${FW_VERSION:-Unknown release} @ $(cat /proc/sys/kernel/hostname)"
+ }
+ 
+ write_lldpd_conf()
+@@ -48,6 +45,10 @@ write_lldpd_conf()
+ 	[ -n "$lldp_description" ] && echo "configure system description" "\"$lldp_description\"" >> "$LLDPD_CONF"
+ 	[ -n "$lldp_hostname" ] && echo "configure system hostname" "\"$lldp_hostname\"" >> "$LLDPD_CONF"
+ 
++	# Prevent br-lan IP from being used for LLDP
++	echo "configure system ip management pattern !192.168.1.1" >> "$LLDPD_CONF"
++	echo "configure system ip management pattern !192.168.0.1" >> "$LLDPD_CONF"
```

Comment:
Not sure what this is doing.  There is some possibility that if Lan is 192.168.0.1 then the Wan might be 192.168.1.1 or vice versa.  Let's discuss.

Reply:
Should be able to specify an interface eg br-wan or wan.  Per https://manpages.debian.org/testing/lldpd/lldpcli.8.en.html

Reply:
Did that, had to update lldpd to 1.0.5 for that feature to work though https://github.com/lldpd/lldpd/releases/tag/1.0.5